public/docs/_examples/hierarchical-dependency-injection/ts/src/app/hero-tax-return.component.ts

```diff
@@ -15,10 +15,11 @@ import { HeroTaxReturnService } from './hero-tax-return.service';
 export class HeroTaxReturnComponent {
   message = '';
   @Output() close = new EventEmitter<void>();
-  @Input()
+
   get taxReturn(): HeroTaxReturn {
     return this.heroTaxReturnService.taxReturn;
   }
+  @Input()
```

Comment:
I don't feel strongly about it  ... yet.

If we really believe that `@Input` should adorn the setter, we should find a place in the docs to say that and say why. If you have a moment to look for that place and propose language for it, I'd be grateful.

That would be a separate PR. I'm merging this one.